blueprints/frost-component/files/__root__/__path__/__name__.js

```diff
@@ -1,13 +1,34 @@
 import Ember from 'ember'
-import FrostEvents from 'ember-frost-core/mixins/frost-events'
-import {
-  PropTypes
-} from 'ember-prop-types'
+const { Component } = Ember
+import PropTypesMixin, { PropTypes } from 'ember-prop-types'
+import layout from '<%= templatePath %>'
 
-const {
-  Component
-} = Ember
+const <%= classifiedModuleName %> = Component.extend(PropTypesMixin, {
```

Comment:
Correct, if you rely on the initializer then `getDefaultProps()` isn't run in your integration tests and you start wondering why the hell props don't have their respective default values. I've spent hours around this and never found a way to easily auto-run an initializer for integration tests and quickly decided it isn't worth the pain.